chapters/pre-recorded-stt/speaker-diarization.mdx

```diff
@@ -87,7 +90,7 @@ Speakers will be assigned indexes by **order of appearance** (i.e. the 1st speak
 
 ## Improving diarization accuracy
 
-<Tip> The following parameters are only available for the `diarization` model. `diarization_enhanced` does not support those yet. </Tip>
+<Tip> The following parameters are only available for the `diarization` model. `enhanced diarization` does not support those yet. </Tip>
```

Comment:
"The following parameters are only available when `diarization_config.enhanced` is `false`. Enhanced diarization does not support those yet." (backticks removed) or something like that.